lib/stellar_spectrum/services/log.rb

```diff
@@ -0,0 +1,13 @@
+module StellarSpectrum
+  class Log
+
+    def self.write(message)
+      logger = StellarSpectrum.configuration.logger
+
+      if logger
+        logger.warn(message)
```

Comment:
And then here, like mentioned above, is where we put the `LOG_TAG` (maybe the constant should be moved to this class then).

Reply:
Yes, I agree that's better!